modular_splurt/code/modules/jobs/job_types/_job_alt_titles.dm

```diff
@@ -315,6 +315,14 @@
 	LAZYADD(alt_titles, extra_titles)
 	. = ..()
 
+/datum/job/peacekeeper/New()
+	var/list/extra_titles = list(
+		"Hired Muscle",
+		"Bodyguard"
+	)
+	LAZYADD(alt_titles, extra_titles)
+	. = ..()
```

Comment:
you can just add these directly to the list in the job file since it's a splurt job

Reply:
I somehow missed that. Thank you for informing me.